internal/controllers/gateway/route_utils_test.go

```diff
@@ -220,6 +219,10 @@ func TestFilterHostnames(t *testing.T) {
 	}
 }
 
+func addressOf[T any](v T) *T {
+	return &v
+}
```

Comment:
Please use https://github.com/Kong/kubernetes-ingress-controller/blob/2a868f3c056b356ead9b286435d9b53f11196a88/internal/util/address/of.go#L3-L6

Reply:
rebasing to main bought this.